packages/lib-classifier/src/store/feedback/strategies/survey/index.spec.js

```diff
@@ -0,0 +1,31 @@
+import chai from 'chai'
+
+import index from './index'
+
+const expect = chai.expect
+
+describe('Feedback > Survey > Index', function () {
+  it('should have a `createRule` property', function () {
+    expect(index.createRule).to.be.a('function')
+  })
+
+  it('should have a `id` property', function () {
+    expect(index.id).to.equal('survey')
+  })
+
+  it('should have a `title` property', function () {
+    expect(index.title).to.equal('Survey')
+  })
+
+  it('should have a `labComponent` property', function () {
+    expect(index.labComponent).to.equal(null)
```

Comment:
Non-blocking. This can use `null()` 
```suggestion
    expect(index.labComponent).to.be.null()
```